docs/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@nuxtjs/feed-next-docs",
+  "version": "1.0.0",
+  "description": "Docs for @nuxtjs/feed-next",
+  "author": "Maciej Pędzich",
+  "scripts": {
+    "dev": "nuxt",
+    "build": "nuxt build",
+    "start": "nuxt start",
+    "generate": "nuxt generate"
+  },
+  "dependencies": {
```

Comment:
Should be devDependency.

Reply:
I don't understand this one either, it's totally valid IMO